packages/flutter/lib/src/rendering/viewport.dart

```diff
@@ -297,7 +296,7 @@ abstract class RenderViewportBase<ParentDataClass extends ContainerParentDataMix
   /// {@endtemplate}
   double get cacheExtent => _cacheExtent;
```

Comment:
replace from the getter until the assert:
```dart
  ///
  /// The getter can never return null, but the field is nullable
  /// because the setter can be set to null to reset the value to
  /// [RenderAbstractViewport.defaultCacheExtent] (in which case
  /// [cacheExtentStyle] must be [CacheExtentStyle.pixel]).
  ///
  /// See also:
  ///
  ///  * [cacheExtentStyle], which controls the units of the [cacheExtent].
  double? get cacheExtent => _cacheExtent;
  double _cacheExtent;
  set cacheExtent(double? value) {
    value ??= RenderAbstractViewport.defaultCacheExtent;
```